admin/spec/controllers/spree/admin/orders_controller_spec.rb

```
@@ -205,17 +205,31 @@ def update_payment_state_after_refund(order)
+
+    context 'when payment method is destroyed' do
+      let(:payment_method) { create(:credit_card_payment_method) }
+      let!(:payment) { create(:payment, payment_method: payment_method, source: credit_card, order: order, amount: order.total) }
+      let!(:credit_card) { create(:credit_card, payment_method: payment_method) }
+
+      it 'shows an order' do
+        subject
+        expect(response).to render_template(:edit)
+        expect(assigns[:payments]).to contain_exactly(payment)
+      end
+    end
```

RuboCop (1.76.1)

[convention] Example group has too many memoized helpers [7/5]

(RSpec/MultipleMemoizedHelpers)